### DISCUSSION TITLE
Installations Image

### DISCUSSION THREAD

**Discussion by Hottihotte** [ramensoftware/windhawk-mods#1181] [General]
How can you add Windhawks including certain mods and settings to an installation medium?
Windhawk is installed via firstlogon.cmd using the command
`winget install RamenSoftware.Windhawk --accept-package-agreements --accept-source-agreements --silent`
I have already introduced the folder and registry in the install.wim.
`%ProgramData%\Windhawk\Engine\Mods
 %ProgramData%\Windhawk\ModsSource
 HKLM\SOFTWARE\Windhawk\Engine\Mods
 HKLM\SOFTWARE\Windhawk\Engine\ModsWritable`
But the mods are not active after installation

**Comment by m417z** [comment]
An export/import feature is planned, which will help in this case. Meanwhile, there's no official way to do this. The comments here and in the lined discussion may help: https://github.com/ramensoftware/windhawk/issues/195.

**Comment by Hottihotte** [comment]
works great
Thank you
But two more questions
Taskbar height and icon size, why is the start icon so far from the edge with icon size: 16x16, taskbar height: 34, even with icon size: 16x16, taskbar height: 34, taskbar button width: 28
Windows 11 Start Menu Styler how can you make the page for pinned apps narrower when no apps are pinned?

**Reply by m417z** [reply]
> Taskbar height and icon size, why is the start icon so far from the edge with icon size: 16x16, taskbar height: 34, even with icon size: 16x16, taskbar height: 34, taskbar button width: 28

Please post a screenshot to show what you mean. Generally, if you want specific adjustments, you can use Windows 11 Taskbar Styler.

> Windows 11 Start Menu Styler how can you make the page for pinned apps narrower when no apps are pinned?

I didn't try doing something like that. You can try asking here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues

**Reply by Hottihotte** [reply]
> Please post a screenshot to show what you mean. Generally, if you want specific adjustments, you can use Windows 11 Taskbar Styler.

It's been resolved, it was a mistake on my part.
In Windows 11, the Start button is generally shifted a little further to the right than in Windows 10.
